neqo-transport/src/connection.rs

```diff
@@ -1137,7 +1149,13 @@ impl Connection {
             let mut has_padding = false;
             let cong_avail = match self.tx_mode {
                 TxMode::Normal => usize::try_from(self.loss_recovery.cwnd_avail()).unwrap(),
-                TxMode::Pto => path.mtu(), // send one packet
+                TxMode::Pto { dgram_avail, .. } => {
```

Comment:
Why are we saving this information on the connection?  Can't the loss recovery spaces hold their PTO state?

Let's say that we send PN 3 in Handshake, can't we ask the loss recovery whether it is time to PTO in that space when it comes time to send a packet in that space?

Reply:
I am not sure either whether we need tx_mode. I think this can be done in a different way.
Also our way of calling 'process' multiple times does not fit nicely with PTO for example. 
dgram_avail is number of packet available to be sent. it should be 1 or 2, but setting it to 2 doesn't work with the way we build frames and packet. I will write a long comment in the e-mail thread you have started.